README.md

```diff
@@ -1,8 +1,36 @@
 # BlaBlaCar React component library
 
-## CLI
-There is an internal set of commands available, to bkirk. To bootstrap our new componentst:
+## Installation
+BlaBlaCar `ui-library` is available as an [npm package](https://www.npmjs.com/package/@blablacar/ui-library)
 
+Just run this inside your project
 ```
-npm run kirk
+npm i @blablacar/ui-library
```

Comment:
Maybe you should also add yarn to seem fancy

Reply:
:trollface: